huaweicloud/services/eps/resource_huaweicloud_enterprise_project.go

```diff
@@ -108,11 +107,11 @@ func resourceEnterpriseProjectCreate(ctx context.Context, d *schema.ResourceData
 }
 
 func resourceEnterpriseProjectRead(_ context.Context, d *schema.ResourceData, meta interface{}) diag.Diagnostics {
-	config := meta.(*config.Config)
-	epsClient, err := config.EnterpriseProjectClient(config.GetRegion(d))
+	conf := meta.(*config.Config)
+	epsClient, err := conf.EnterpriseProjectClient(conf.GetRegion(d))
 
 	if err != nil {
-		return fmtp.DiagErrorf("Unable to create HuaweiCloud EPS client : %s", err)
+		return diag.Errorf("Unable to create EPS client : %s", err)
```

Comment:
return diag.Errorf("Unable to create EPS client: %s", err)

Reply:
pls check for other similar errors.

Reply:
fixed